smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/TypeScriptIntegration.java

```diff
@@ -68,64 +55,6 @@ default Model preprocessModel(Model model, TypeScriptSettings settings) {
         return model;
```

Comment:
Is this method necessary still?

Reply:
Still have to move SSDK codegen to DirectedCodegen, so currently it is used in CodegenVisitor. Once I move SSDK to DirectedCodegen too, will delete CodegenVisitor and this method.

Reply:
Oh, I was thinking of something else. You are right - the `preprocessModel` method with default implementation is already present in `SmithyIntegration` so not needed here.